gammapy/catalog/fermi.py

```diff
@@ -483,16 +492,136 @@ class SourceCatalog2FHL(SourceCatalog):
 
     def __init__(self, filename=None):
         if not filename:
-            filename = gammapy_extra.filename('datasets/catalogs/fermi/gll_psch_v08.fit.gz')
+            filename = gammapy_extra.filename(
+                'datasets/catalogs/fermi/gll_psch_v08.fit.gz')
 
         self.hdu_list = fits.open(filename)
         self.count_map_hdu = self.hdu_list['Count Map']
-        self.extended_sources_table = Table(self.hdu_list['Extended Sources'].data)
+        self.extended_sources_table = Table(
+            self.hdu_list['Extended Sources'].data)
         self.rois = Table(self.hdu_list['ROIs'].data)
         table = Table(self.hdu_list['2FHL Source Catalog'].data)
 
-        source_name_key='Source_Name'
+        source_name_key = 'Source_Name'
         source_name_alias = ('ASSOC', '3FGL_Name', '1FHL_Name', 'TeVCat_Name')
         super(SourceCatalog2FHL, self).__init__(table=table,
-                                source_name_key=source_name_key,
-                                source_name_alias=source_name_alias)
+                                                source_name_key=source_name_key,
+                                                source_name_alias=source_name_alias)
+
+
+class SourceCatalogObject1FHL(SourceCatalogObject):
+    """One source from the Fermi-LAT 1FHL catalog.
+    """
+    _ebounds = EnergyBounds([10, 30, 100, 500], 'GeV')
+    _ebounds_suffix = ['10_30', '30_100', '100_500']
+    energy_range = Quantity([0.01, 0.5], 'TeV')
+    """Energy range of the Fermi 1FHL source catalog"""
+
+    def __str__(self):
+        """Print summary info."""
+        # TODO: can we share code with 3FGL summary funtion?
```

Comment:
Yeah, I don't know.

My guess would be that code re-use will be limited, and one could just write this pretty-print output top to bottom like we did in the HGPS source class.
If no-one has time for now, an alternative would be to put this at the end:
```
from pprint import pprint
pprint(self.data)
```
so that at least all info is printed, one item per line.
Or put it into an Astropy table with two columsn "Parameter" and "Value" and call pprint on that.

In practice this is really nice to have as you look around the catalogs, and should be added to the docs.

But also, mostly unrelated to your PR here, so OK to just keep as-is, unless you want to add this convenience.